### DISCUSSION TITLE
Trackpad Two Finger Scroll Support

### DISCUSSION THREAD

**Discussion by emreozcelik** [ClassicOldSong/moonlight-android#1] [Q&A] [answered]
I couldn't get the trackpad scrolling to work on Tab S9. I also couldn't find an option for it in the settings. Am I missing something, or is this issue specific to the Tab S9?

**Comment by ClassicOldSong** [comment]
Does tap as click work on your tablet?

Also can you check if Trackpad(Natural) mode works on your touch screen

Trackpad support is enabled by default and there's no option for it.

**Comment by ClassicOldSong** [comment, marked as answer]
@emreozcelik let's move to issues as discussions won't get me emailed with new discussions and replies
